ota_university/settings.py

```diff
@@ -57,9 +57,15 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'ctf_framework',
-    'django_slack_oauth'
+    'django_slack_oauth',
+    'rules.apps.AutodiscoverRulesConfig'
 ]
 
+AUTHENTICATION_BACKENDS = (
+    'rules.permissions.ObjectPermissionBackend',
+    'django.contrib.auth.backends.ModelBackend',
```

Comment:
So what exactly is this ModelBackend?

Reply:
Still have a hard time understanding it... It's essential for authentication though, and its linked to the User class afaik.